Online/Menus/VSScreen/InitVsSplash.asm

```diff
@@ -282,6 +283,7 @@ b EXIT
 ################################################################################
 INIT_PLAYER_TEXT:
 backup
+fbackup 1
```

Comment:
This file uses 31 and 30

Reply:
@JLaferri the reg var is being set to 30, but I never use it.

Reply:
Please be more thorough with your evaluation, especially when I mention it.
https://github.com/project-slippi/slippi-ssbm-asm/blob/1a583909d2619873d727d78435a26cd3745d5bb7/Online/Menus/VSScreen/InitVsSplash.asm#L301

Reply:
You are correct, apologies.